openmdao/docs/openmdao_book/features/core_features/working_with_components/distributed_components.ipynb

```diff
@@ -37,45 +37,21 @@
    "cell_type": "markdown",
```

Comment:
<p>Suggest: In the second paragraph of the note explaining Auto-IVCs, maybe you should call out that definition of the term ("Auto-IVC") since it is used in the earlier discussion...</p><p> </p><p>Should "matrix-free format" have a hyperlink to the relevant documentation explaining what it is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Added definition of Auto-IVC, and added a link directly to the matrix-free example.</p>